data/tools/wmllint

```diff
@@ -1288,6 +1288,8 @@ linechanges = (
         ("portraits/monsters/sea-serpent.png", "portraits/monsters/sea-serpent.webp"),
         ("portraits/monsters/big-icemonax.png", "portraits/monsters/big-icemonax.webp"),
         ("portraits/monsters/ogre.png", "portraits/monsters/ogre.webp"),
+        # introduced in wesnoth 1.17.18
```

Comment:
I think this is a little confusing since it's not in 1.17.18. Should either say `1.17.19` or `after`.

Reply:
![image](https://github.com/wesnoth/wesnoth/assets/5283677/dfdf1b86-ee25-4c5c-946e-77f90d08508d)
Oops...I see that it is x.x.19